pytrainer/extensions/osm.py

```diff
@@ -3,20 +3,18 @@
 # TODO: Add Google satellite images layers ?
 # TODO: Create map class/interface that osm/googlemaps will inherit from ?
 
-import os
-import re
+import html
 import logging
-import requests         # for downloading cached versions of openlayers.js and openstreetmaps.js
-import time             # Used for checking if local cached file is current
+import os
+import time  # Used for checking if local cached file is current
 
-import cgi as html
+import requests  # for downloading cached versions of openlayers.js and openstreetmaps.js
```

Comment:
After this change it's still unclear why the `import requests` was moved. I would expect it to be in the same place as before, between `o` and `t` alphabetically.

I think the indentations for the comments fulfilled a purpose too.

Reply:
There are three kinds of imports. Stdlib, third-party and local. Requests belongs to the second group, and thus split of from stlib imports.

Reply:
Ah, if the indentation changes was intentional then everything looks fine now.